ts/components/conversation/H5AudioPlayer.tsx

```diff
@@ -171,7 +171,7 @@ export const AudioPlayerWithEncryptedFile = (props: {
     direction === 'incoming'
       ? 'var(--message-bubbles-received-text-color)'
       : 'var(--message-bubbles-sent-text-color)';
-  const dataTestId = `audio-${messageId}`;
+  const dataTestId: SessionDataTestId = 'audio-player';
```

Comment:
You can have multiple audio players on screen do we not want the test id to be unique?

Reply:
This is only used for QA. And we have other elements which have duplicates. I guess that's alright.